src/cli-options.ts

```diff
@@ -301,7 +301,7 @@ As a comparison: without using the --peer option, ncu will suggest the latest ve
   {
     long: 'registry',
     short: 'r',
-    arg: 'url',
+    arg: 'url' || 'path',
```

Comment:
`arg` must be a single argument name, to be used by commander. (As written this will always resolve to `url`, since the string literal `'url'` is always truthy.)

I would suggest we change this to `uri`. Even though we don't use an explicit [file URI scheme](https://en.wikipedia.org/wiki/File_URI_scheme), both urls and files fit conceptually into a "URI".